Resources/public/js/components/bulk-price/bulk-price.html

```diff
@@ -43,4 +43,47 @@ <h2 class="divider"><%= currency.name %></h2>
             </table>
         </div>
     </div>
+    <div class="grid-row small">
+        <div class="grid-col-6">
+            <div class="form-group">
+                <label><%= translate('product.bulkprice.specialprice') %></label>
+                <input id="<%= !!specialPrice.dateId.inputId ? specialPrice.dateId.inputId : '' %>" class="form-element husky-validate specialprice align-right" type="text"  value="<%= specialPrice.price %>"/>
+            </div>
+        </div>
+        <div class="grid-col-6">
+            <div class="form-group">
+                <label><%= translate('product.bulkprice.validfrom') %></label>
+                <div data-form="true"
+                     data-type="husky-input"
+                     data-aura-component="input@husky"
+                     data-aura-skin="date"
+                     data-aura-instance-name="<%= !!specialPrice ? specialPrice.dateId.from : '' %>"
```

Comment:
remove all !! 

this is not needed in if causes, as the you are checking if the property exists nor is empty. 
so as null, undefined, none, false etc ... would all be false it is not needed